zip-codes/scripts/map.js

```diff
@@ -1,12 +1,17 @@
 'use strict';
 
-function initMap() {
+function initMap(myLatlng) {
   // Create a map object and specify the DOM element for display.
   var map = new google.maps.Map(document.getElementById('map'), {
     center: {lat: 47.611435, lng: -122.330456},
     scrollwheel: true,
     zoom: 8
   });
 
+  var marker = new google.maps.Marker({
+    position: myLatlng,
+    map: map
+  });
+
   // TODO: Follow the Google Maps API docs to create markers on the map based on the search options on the home page.
```

Comment:
You'll need a function to call to place a marker on the map when you use the filter boxes instead of the zip code form submission :)